hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/SqlExecuteCodec.java

```diff
@@ -103,6 +105,10 @@ public static ClientMessage encodeRequest(java.lang.String sql, java.util.Collec
 
     @edu.umd.cs.findbugs.annotations.SuppressFBWarnings({"URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"})
     public static class ResponseParameters {
+        /**
+         * True, if this result is an update count, false, if it contains rows.
+         */
+        public boolean isUpdateCount;
```

Comment:
On the protocol level, we may encode the missing update count as `-1` and thus avoid this flag.